pandas/core/strings/object_array.py

```diff
@@ -435,7 +435,7 @@ def _str_rstrip(self, to_strip=None):
 
     def _str_removeprefix(self, prefix: str) -> Series:
         # outstanding question on whether to use native methods for users
-        # on Python 3.9+ https://git.io/JE9QK, in which case we could do
+        # on Python 3.9+ https://bit.ly/3LuMeRn, in which case we could do
```

Comment:
@charlogazzo mind actually updating this to the full link? https://github.com/pandas-dev/pandas/pull/39226#issuecomment-836719770